std/parallelism.d

```diff
@@ -2633,7 +2633,7 @@ public:
 
             // Hack to take the address of a nested function w/o
             // making a closure.
-            static auto scopedAddress(D)(scope D del)
+            static auto scopedAddress(D)(scope D del) @system
```

Comment:
As far as I understood it Walter wants to make dirty work where a `scope` can escape only valid if the user tells the compiler to do so explicitly to avoid common mistakes and foster the `scope` experience.